infrastructure/docker-compose.deploy.yml

```diff
@@ -15,12 +15,8 @@ services:
   traefik:
     image: 'traefik:v2.9'
     ports:
-      - target: 80
-        published: 80
-        mode: host
-      - target: 443
-        published: 443
-        mode: host
+      - '${VPN_HOST_ADDRESS:-0.0.0.0}:80:80'
+      - '${VPN_HOST_ADDRESS:-0.0.0.0}:443:443'
```

Comment:
I think we need to move this to docker-compose-vpn.deploy.yml.  Then deploy.sh script should pick that up only if we are provisioning a wireguard VPN

Reply:
This should work for all VPN and non-vpn setups. If VPN_HOST_ADDRESS is not set then it defaults to all ip addresses (0.0.0.0)

Reply:
Of course!